src/Tester/Tester.csproj

```diff
@@ -43,62 +43,67 @@
     <AssemblyOriginatorKeyFile>..\Orleans.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Bond">
-      <HintPath>$(SolutionDir)packages\Bond.Runtime.CSharp.3.0.7\lib\net45\Bond.dll</HintPath>
```

Comment:
In fairness to @attilah, i also want to say that we do follow a few slightly "non-standard" process for package references in Orleans Project that the NuGet tool does not follow, but which we haven't clearly written down except as anecdotes. 

That is partly my-bad, and I have taken an action item to fill that gap [see my reply to @jason-bragg above]